sgkit/distance/api.py

```diff
@@ -135,7 +162,7 @@ def _aggregate(x_chunk: ArrayLike, **_: typing.Any) -> ArrayLike:
         producing the final output. It is always invoked, even when the reduced
         Array counts a single chunk along the reduced axes."""
         x_chunk = x_chunk.reshape(x_chunk.shape[:-2] + (-1, n_map_param))
```

Comment:
Can you add some comments on what the `x_chunk` dimensions are coming into this function?